app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -0,0 +1,57 @@
+module Users
+  class PivCacAuthenticationSetupController < ApplicationController
+    include UserAuthenticator
+
+    before_action :confirm_two_factor_authenticated
+    before_action :authorize_piv_cac_setup, only: :new
+    before_action :authorize_piv_cac_disable, only: :disable
+
+    def new
+      analytics.track_event(Analytics::USER_REGISTRATION_PIV_CAC_SETUP_VISIT)
+      if params.key?(:token)
+        process_token(params[:token])
+      else
+        render :new
+      end
+    end
+
+    def disable
```

Comment:
What do you think about calling this method `delete` to adhere to the Rail REST convention?

Reply:
Yeah, I just noticed that. I think we should use the usual `create`, `new`, `show`, `delete` actions and then go back and update the older controllers to keep things consistent.